packages/react/select/src/Select.tsx

```diff
@@ -380,6 +380,7 @@ const SelectContentImpl = React.forwardRef<SelectContentImplElement, SelectConte
     const [isPositioned, setIsPositioned] = React.useState(false);
     const shouldRepositionRef = React.useRef(true);
     const shouldExpandOnScrollRef = React.useRef(false);
+    const firstItemRef = React.useRef(false);
```

Comment:
As we're using this to short-circuit perhaps something like `hasFoundFirstSelectableItemRef` (or a less verbose alternative) might be better? At first glance this reads like it was the `ref` of the first item regardless of validity.

Also, is this something we need to cleanup after closing?

Reply:
How about `firstValidItemFoundRef`?